### PR TITLE
Add API Key QR scan + click to reveal api key in List view

### PR DESCRIPTION
**Pull request by Kukks**
I did not make use of the new BCUR animated QR standard since this is BTCPay specific and the data being set is of minimal length but I can easily switch it over.

@21isenough 
image

### CODE REVIEW

**Comment by 21isenough** [comment]
Awesome :smiley: :smiling_face_with_three_hearts:

**Comment by Kukks** [comment]
> Awesome 😃 🥰

Do you prefer this or a BCUR encoded QR code?

**Comment by 21isenough** [comment]
No, this is great. As there is not much data, this should suffice. 

Am I correct that user and password would only be needed for the "Basic HTTP authentication ", right? 
Once I have the API key acquired the key and the URL are enough, I assume. So this is perfect :+1:

**Comment by Kukks** [comment]
> No, this is great. As there is not much data, this should suffice.
> 
> Am I correct that user and password would only be needed for the "Basic HTTP authentication ", right?
> Once I have the API key acquired the key and the URL are enough, I assume. So this is perfect 👍

Yeah, correct. There is also an endpoint to check that the api key you are using has sufficient permissions for you

File: BTCPayServer/Views/Shared/ShowQR.cshtml [hunk 1]
```diff
@@ -78,7 +79,11 @@
  					this.fragments = [];
  					return;
  				}
- 				this.fragments = window.bcur.encodeUR(this.data.toString(), 200);
+ 				if (this.useBCUR){
```
Comment (NicolasDorier) [inline BTCPayServer/Views/Shared/ShowQR.cshtml:82]: what is BCUR?

Comment (Kukks) [reply BTCPayServer/Views/Shared/ShowQR.cshtml:82]: BCUR is the spec used for animated QR codes (we use it for psbt, wallet import, wallet export) 
https://github.com/BlockchainCommons/bc-ur

Not used in this PR

**Pull request merged**